### PR TITLE
[dvsim] Better dashboard result for parameterized blocks

### PR DESCRIPTION
**Pull request by weicaiyang**
It showed the same name when verifying 2 types sram. This fixed it.

Now it shows like this for kmac and sram.
|                                       Name                                       |  Passing  |  Total  |  Pass Rate  |
|:--------------------------------------------------------------------------------:|:---------:|:-------:|:-----------:|
|            [KMAC/MASKED](../../../../ip/masked/dv/latest/report.html)            |     3     |    3    |  100.00 %   |
|          [KMAC/UNMASKED](../../../../ip/unmasked/dv/latest/report.html)          |     3     |    3    |  100.00 %   |
|           [SRAM_CTRL/MAIN](../../../../ip/main/dv/latest/report.html)            |     3     |    3    |  100.00 %   |
|            [SRAM_CTRL/RET](../../../../ip/ret/dv/latest/report.html)             |     3     |    3    |  100.00 %   |

Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

**Review by marnovandermaas — APPROVED** [review]
LGTM

**Review by cindychip — APPROVED** [review]
Thanks weicai

File: util/dvsim/SimCfg.py [hunk 1]
```diff
@@ -122,6 +123,7 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
         self.run_cmd = ""
 
         # Generated data structures
+        self.full_name = ""
```
Comment (sriyerg) [inline util/dvsim/SimCfg.py:126]: Could we set the `name` in `kmac_*_sim_cfg.hjson` to `kmac_masked` and remove `name` from `kmac_base_sim_cfg.hjson` (and likewise for sram)? I feel that may be an easier fix.

Comment (weicaiyang) [reply util/dvsim/SimCfg.py:126]: we need a `name` without the variant.
The csr tests use `{name}_common_vseq`. We only have kmac_common_vseq. With your suggestion, we need to have `kmac_masked_common_vseq` and `kmac_unmask_common_vseq`. Likewise for other common tests.
Any thoughts?

**Pull request merged**